readme.md

```diff
@@ -2,109 +2,106 @@
 
 ### Overview
 
-You’ve already worked in small groups to accomplish various labs and exercises, but this time **we’re going to challenge you to work on a whole project with a small team.**
-
-Not only will you be asked to **exercise additional creativity** in designing your own project, your instructors will partner you with other classmates to architect, design, and collaboratively build an API of your own design.
-
 While your last project taught you how to get started with Ruby, SQL, & Ruby on Rails, this project you'll be building something exciting with **Express & Sequelize.**
 
-**This is meant to push you both technically and collaboratively.** It’s a lot harder to work in a team than to work by yourself, but that's most likely you’re going to find yourself doing in your first development job after WDI, and **it's important to learn how to work together.**
```

Comment:
The group dynamics *can* be harder, but I've found it to be easier to work in a team and far more enjoyable.  Creating a PR to reword this.  This phrase has a grammar issue too "that's most likely you're".